doc/examples/flare/lib/main.dart

```diff
@@ -20,10 +20,11 @@ void main() {
 }
 
 class MyGame extends BaseGame {
-  final TextConfig fpsTextConfig = TextConfig(color: const Color(0xFFFFFFFF));
+  final TextConfig fpsTextConfig =
+      const TextConfig(color: const Color(0xFFFFFFFF));
 
-  final paint = Paint()..color = Color(0xFFe5e5e5e5);
-  List<String> _animations = ["Stand", "Wave", "Jump", "Dance"];
+  final paint = Paint()..color = const Color(0xFFe5e5e5e5);
```

Comment:
Since you changed this line, make the color upper case

Reply:
that should be on our linter!!!